tests/prompt_adapter/test_pa_lora.py

```diff
@@ -0,0 +1,60 @@
+from huggingface_hub import snapshot_download
+
+from vllm import EngineArgs, LLMEngine, SamplingParams
+from vllm.lora.request import LoRARequest
+from vllm.prompt_adapter.request import PromptAdapterRequest
+
+MODEL_PATH = "meta-llama/Llama-2-7b-hf"
+pa_path = snapshot_download(repo_id="swapnilbp/llama_tweet_ptune")
+lora_path = snapshot_download(repo_id="yard1/llama-2-7b-sql-lora-test")
+
+
+def do_sample(engine):
+
+    prompts = [
+        (
+            'Tweet text : @nationalgridus I have no water and the bill is current and paid. Can you do something about this? Label :',  # noqa: E501
+            SamplingParams(temperature=0.0, max_tokens=3, stop_token_ids=[3]),
+            PromptAdapterRequest("hate_speech", 1, pa_path, 8),
+            LoRARequest("sql_test", 1, lora_path)),
+        (
+            "[user] Write a SQL query to answer the question based on the table schema.\n\n context: CREATE TABLE table_name_74 (icao VARCHAR, airport VARCHAR)\n\n question: Name the ICAO for lilongwe international airport [/user] [assistant]",  # noqa: E501
+            SamplingParams(temperature=0.0,
+                           max_tokens=100,
+                           stop=["[/assistant]"]),
+            PromptAdapterRequest("hate_speech", 1, pa_path, 8),
+            LoRARequest("sql_test", 1, lora_path)),
+    ]
+
+    request_id = 0
+    results = set()
+    while prompts or engine.has_unfinished_requests():
+        if prompts:
+            prompt, sampling_params, pa_request, lora_request = prompts.pop(0)
+            engine.add_request(str(request_id),
+                               prompt,
+                               sampling_params,
+                               prompt_adapter_request=pa_request,
+                               lora_request=lora_request)
+            request_id += 1
+
+        request_outputs = engine.step()
+
+        for request_output in request_outputs:
+            if request_output.finished:
+                results.add(request_output.outputs[0].text)
+    print(results)
+    return results
+
+
+def test_lora_prompt_adapter():
```

Comment:
can we add a test where we have a dummy prompt embedding (which should have no effect on the output) and ensure that lora with dummy prompt embedding gives the same output as lora by itself?